client/pkg/omni/resources/omni/ongoing_task.go

```diff
@@ -40,8 +40,7 @@ type OngoingTaskExtension struct{}
 func (OngoingTaskExtension) ResourceDefinition() meta.ResourceDefinitionSpec {
 	return meta.ResourceDefinitionSpec{
 		Type:             OngoingTaskType,
-		Aliases:          []resource.Type{},
-		DefaultNamespace: resources.DefaultNamespace,
+		DefaultNamespace: resources.EphemeralNamespace,
```

Comment:
Good change, but nothing will drop the ongoing task resources from the etcd state.

Reply:
oh, I see